tests/Connectors/Integration/Stripe/CompanySubscriptionTest.php

```diff
@@ -0,0 +1,161 @@
+<?php
+
+declare(strict_types=1);
+
+namespace Tests\Connectors\Integration\Stripe;
+
+use Illuminate\Support\Facades\DB;
+use Kanvas\Apps\Models\Apps;
+use Kanvas\Companies\Models\Companies;
+use Kanvas\Connectors\Stripe\Enums\ConfigurationEnum;
+use Kanvas\Subscription\Plans\Models\Plan;
+use Laravel\Cashier\Subscription;
+use Tests\TestCase;
+
+final class CompanySubscriptionTest extends TestCase
+{
+    protected Companies $company;
+    protected Apps $appModel;
+    protected string $paymentMethodId;
+    protected Plan $plan;
+    protected $price;
+
+    protected function setUp(): void
+    {
+        parent::setUp();
+        $this->company = Companies::factory()->create();
+        $this->appModel = app(Apps::class);
+        if (empty($this->appModel->get(ConfigurationEnum::STRIPE_SECRET_KEY->value))) {
+            $this->appModel->set(ConfigurationEnum::STRIPE_SECRET_KEY->value, getenv('TEST_STRIPE_SECRET_KEY'));
+        }
+        $this->paymentMethodId = $this->createPaymentMethod();
+        $this->seedAppPlansPrices();
+        $this->plan = Plan::where('apps_id', $this->appModel->getId())->firstOrFail();
+        $this->price = $this->plan->price()->firstOrFail();
+    }
+
+    protected function seedAppPlansPrices()
+    {
+        // Define the data you want to insert
+        $prices = [
+            [
+                'apps_plans_id' => 1,
+                'stripe_id' => 'price_1Q11XeBwyV21ueMMd6yZ4Tl5',
```

Comment:
This id, I believe that is move to env, @kaioken can be add this id to the test env